### PR TITLE
dark theme add (optional)

### PR DESCRIPTION
**Pull request by vitaly-zdanevich**
CSS adapt for system or browser settings (light or dark).

### CODE REVIEW

**Comment by vitaly-zdanevich** [comment]
FJIMG_20201111_224756

Now I can read it at night.

File: style-additions.css [hunk 1]
```diff
@@ -25,3 +25,51 @@ td.repo {
   width: 100%;
   border-bottom: 2px solid #ddd;
 }
+
+@media (prefers-color-scheme: dark) {
```
Comment (limitedmage) [inline style-additions.css:29]: Can you please remove the `!impotant` here? 
- For inline styles (eg. `.page-header`), edit the HTML to remove the inline styles and move them over to this CSS file
- For bootstrap classes (eg. `.success`) you can make the selector more specific

Comment (vitaly-zdanevich) [reply style-additions.css:29]: Why do we need more specific selectors?

Comment (limitedmage) [reply style-additions.css:38]: So that the styles are overwritten without the need for `!important`

File: style-additions.css [hunk 2]
```diff
@@ -25,3 +30,55 @@ td.repo {
   width: 100%;
   border-bottom: 2px solid #ddd;
 }
+
+.text-center {
+    font-size:24px;
```
Comment (felixfbecker) [inline style-additions.css:35]: **question**: What's this for? It seems weird that a class for centering text would also influence font size?

File: style-additions.css [hunk 3]
```diff
@@ -1,3 +1,8 @@
+.page-header {
+    background-color:#eee;
+    margin-top: 0px;
```
Comment (domainfun) [inline style-additions.css:3]: To keep the CSS clean & lean, may I suggest to remove the "px" here, and further down change "top: 0px;" to "top: 0;" (zero is zero, no matter what unit of measurement).

**Comment by keegancsmith** [comment]
@felixfbecker @limitedmage does this look good to merge?

**Comment by jdorfman** [comment]
This site needs to get updated to the latest version of Bootstrap. @vitaly-zdanevich if you are interested in helping in a new PR I will merge it asap. In the meantime Closing this PR.

**Pull request closed without merging**